src/core/ext/client_config/http_connect_handshaker.c

```diff
@@ -206,11 +206,11 @@ static void http_connect_handshaker_do_handshake(
   handshaker->user_data = user_data;
   handshaker->read_buffer = read_buffer;
   // Send HTTP CONNECT request.
-  gpr_log(GPR_INFO, "Connecting to server %s via HTTP proxy %s",
-          handshaker->server_name, handshaker->proxy_server);
+  gpr_log(GPR_INFO, "Connecting to server %s via HTTP proxy with authority %s",
```

Comment:
I'd still like the log the name of the proxy server, even if we don't use it for anything else.